beta-src/src/components/ui/WDDeleteTracker.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
```

Comment:
WDDeleteTracker.tsx and WDDeleteReminder.tsx are nearly identical. Can we combine them and then we can pass down the props accordingly in WDDeletePanel.tsx